### PR TITLE
Add TPOT recipe

### PR DESCRIPTION
**Pull request by proinsias**
- [x] Add `update_checker` to `conda-forge`
- [x] Tidy up `meta.yaml` after verifying build requirements

### CODE REVIEW

File: recipes/tpot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.9.3
+    - pandas >=0.16.2
+    - scikit-learn >=0.16.1
+    - deap >=1.0
+    - tqdm
+    - update_checker
```
Comment (jakirkham) [inline recipes/tpot/meta.yaml]: Probably can drop everything except `python` and `setuptools` from the build dependencies.

**Comment by proinsias** [comment]
@jakirkham – should be ready to merge???

File: recipes/tpot/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/tpot/meta.yaml]: Done above. Comment appears to have moved.

**Comment by jakirkham** [comment]
Yep, thanks @proinsias. Sorry didn't know this one was done.

**Pull request merged**